### ISSUE TITLE
Accessibility: Review contrast of icons and text in left menu

### ISSUE THREAD

**Issue by GuySartorelli** [silverstripe/silverstripe-admin#1970]
https://github.com/silverstripe/silverstripe-framework/issues/6408 raised some concerns about the contrast of icons in the left menu of the CMS.
Note that when the menu is collapsed, the icons are the only visual indicator of what each section is (without hovering)

At the time of that issue the icons were black - they're now even less contrasting with the background.

One option mentioned in comments was to increase the size of those icons to 20px (they're currently 17px)

## Acceptance criteria

- [x] Have the UX team review the contrast of the text and the icons in the left menu
- [x] Implement any recommended changes

## PRs

- https://github.com/silverstripe/silverstripe-admin/pull/2075

**Comment by cassiezoest** [comment]
So WCAG requires a 3:1 minimum contrast ratio for graphics and UI components. Currently the icons do meet that requirement with the default menu item background, but not with the hover/active background colour. 

### Recommended/possible changes

I would update the icon fill colour to be #497792 which is slightly darker and has sufficient contrast with both states. 

We could bump up the size but 20px feels a touch too big, could do 19px? You might need to adjust some of the padding/margins to make sure everything looks okay after adjusting the size.

### Icons

Obviously we don't really have much control over what icons people use in the sidebar, but I thought it might be worth noting that some icons are better suited for use in the sidebar. Those that are filled/have thicker lines will be easier to distinguish at a smaller size. Some icons in the Silverstripe font are also a bit smaller so icon size isn't always consistent. For example (this is with darker colour + larger icon size): 

<img width="1355" height="893" alt="Image" src="https://github.com/user-attachments/assets/ef090565-d494-45ff-933d-795753713af3" />

I'm pretty sure that the icons which are styled as squares were designed specifically for elemental blocks where the icons are larger, hence thinner lines. Not as good in the sidebar where they're smaller

<img width="456" height="753" alt="Image" src="https://github.com/user-attachments/assets/779ff500-888a-4327-825e-effaca02c9ee" />

### Text

Text contrast is all good.

**Comment by emteknetnz** [comment]
Linked PR merged
